.buildkite/inference-benchmark-pipeline.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -euf
+
+# This script is executed by Buildkite to generate pipeline YAML string.
+# Document: https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines
+#
+# Usage: CONVERT_RESULT_PATH=gs://path/to/result.tgz ./inference-benchmark-pipeline.sh
+
+# test cases to run inference
+# "<buildkite agent-type> <model filename>"
+TEST_CASES=(
+    "de10nano       lib/lib_arm.so"
+    "de10nano       lib/lib_fpga.so"
+
+    "jetson-nano    minimal_graph_with_shape.pb"
+    "jetson-tx2     minimal_graph_with_shape.pb"
+    "jetson-xavier  minimal_graph_with_shape.pb"
+
+    "raspberry-pi   lib/lib_aarch64.so"
+)
+
+# Generate setup step
+cat <<EOS
+steps:
+  - label: "inference: setup"
+    key: "setup"
+    timeout_in_minutes: "30"
+    agents:
+      - "env=benchmark"
+      - "agent-type=gcloudsdk"
+
+    artifact_paths:
+      - "convert-result.tgz"
+
+    command: |
+      gsutil -m cp "${CONVERT_RESULT_PATH}" ./
+EOS
+
+for TEST_CASE in "${TEST_CASES[@]}" ; do
+    IFS=' '
+    read AGENT MODEL <<< "${TEST_CASE}"
+
+    # Generate inference step
+    cat <<EOS
+
+
+  - label: "inference: ${AGENT} (${MODEL})"
+    depends_on: "setup"
+    timeout_in_minutes: "30"
+
+    agents:
+      - "env=benchmark"
+      - "agent-type=${AGENT}"
+
+    artifact_paths:
+      - "export/*/*/output/python/output/output.json"
+
+    command: |
+      buildkite-agent artifact download "convert-result.tgz" ./
```

Comment:
I think we should specify `--build $BUILDKITE_BUILD_ID` to download correct artifacts uploaded by the previous step.
See: https://buildkite.com/docs/agent/v3/cli-artifact#downloading-artifacts

Reply:
Thank you!! Fixed.